### PR TITLE
FE Playwright test: add notes to multiple highlights in content page

### PR DESCRIPTION
**Pull request by Malar-Natarajan**
for: https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/2400

Add notes to multiple highlights in content page and assert them to the text entered

Few lines related to Edit Note are commented out. Edit note will be continued in next PR

### CODE REVIEW

File: playwright/src/fixtures/content.page.ts [hunk 1]
```diff
@@ -165,23 +168,87 @@ class ContentPage {
...
+    // this.colorlocator = await this.colorLocator(color)
+    this.colorlocator = await this.colorLocator(color)
+    const colorLocatorCount = await this.colorlocator.count()
+    if (colorLocatorCount > 1) {
+   
+    for (let i = 0; i < colorLocatorCount; i++) {
+      sleep(4)
+      const colorLocatorVisibility = await this.colorlocator.nth(i).evaluate((e: Element) => {
+        return window.getComputedStyle(e).getPropertyValue('visibility')
+
```
Comment (Malar-Natarajan) [inline playwright/src/fixtures/content.page.ts:184]: @gregfitch I am trying to do some actions on the notecard that is visible in the content page (lines 170 - 190). But the visibility status is returned as `hidden` always in playwright eventhough I see the status `visible` in devtools css layout in debug mode.
I have used similar code above in line 100 and it works without any issue.

Am I missing something in `addNote`

File: playwright/tests/rex-test/rex.behaviorspec.ts [hunk 2]
```diff
@@ -235,3 +237,44 @@ test('note in content page', async ({ page, isMobile }) => {
...
+  const noteText1 = randomstring()
+  await BookPage.addNote(noteText1)
+  // await BookPage.noteConfirmDialog(Actions.Save)
+  
+
+  
+  await BookPage.clickHighlight(0)
+  const noteText0 = randomstring()
+  await BookPage.addNote(noteText0)
+  expect(await BookPage.noteText(noteText1)).toContain(noteText1)
```
Comment (Malar-Natarajan) [inline playwright/tests/rex-test/rex.behaviorspec.ts:276]: @gregfitch can you please look at lines 276 & above. Does it feel like I the expected & actual parameters are the same. The actual value is defined in `content.page.ts` lines 231 - 241

Comment (gregfitch) [reply playwright/tests/rex-test/rex.behaviorspec.ts:276]: it's a bit redundant to expect a value in the object that you've just confirmed exists as the noteText function does its own search (if any text is returned by the function)

Comment (Malar-Natarajan) [reply playwright/tests/rex-test/rex.behaviorspec.ts]: @gregfitch i pushed some changes. when you get time can you please review if the note text verification make sense now.

PS. the pr is still draft

File: playwright/tests/rex-test/rex.behaviorspec.ts [hunk 3]
```diff
@@ -223,15 +223,53 @@ test('note in content page', async ({ page, isMobile }) => {
...
-  await BookPage.editNote(editnoteText)
+  // // AND: Edit note to the highlight and save - Edit Note Update in next PR
+  // const editnoteText = randomstring()
+  // await BookPage.clickContextMenu(0)
+  // await BookPage.editHighlight()
+  // await BookPage.editNote(editnoteText)
+  // await BookPage.noteConfirmDialog(Actions.Save)
+  // expect(await BookPage.noteText()).toBe(editnoteText + noteText)
+})
+
```
Comment (gregfitch) [inline playwright/tests/rex-test/rex.behaviorspec.ts:239]: Do you need this comment?

Comment (Malar-Natarajan) [reply playwright/tests/rex-test/rex.behaviorspec.ts:239]: I am going to update these lines in the next PR. So I prefer this staying here commented.

**Review by gregfitch — APPROVED** [review]

**Pull request merged**